Cronos/cronos-evm-starter-via-eth/project.yaml

```diff
@@ -0,0 +1,55 @@
+specVersion: 1.0.0
+
+name: cronos-evm-starter-via-eth
+version: 0.0.1
+runner:
+  node:
+    name: "@subql/node-ethereum"
+    version: "*"
+  query:
+    name: "@subql/query"
+    version: "*"
+description: "This project can be use as a starting point for developing your Cosmos (Cronos) based SubQuery project"
+repository: "https://github.com/subquery/cosmos-subql-starter/tree/main/Cronos/cronos-evm-starter-via-eth"
+schema:
+  file: ./schema.graphql
+network:
+  chainId: "25"
+  # This endpoint must be a public non-pruned archive node
+  # Public nodes may be rate limited, which can affect indexing speed
+  # When developing your project we suggest getting a private API key
+  # Take a look here for some Cronos endpoints https://moonflow.solutions/rpc-servers
+  endpoint: "https://evm.cronos.org/"
```

Comment:
We should be able to use the same dictionary from the cosmos sdk

Reply:
Ignore that. The dictionary is compatible, but the chainIds wont match. This is something we can improve in the sdk

Reply:
Ok!